### ISSUE TITLE
Package request: emojione-picker

### ISSUE THREAD

**Issue by KiaraGrouwstra** [NixOS/nixpkgs#56455]
## Issue description

URL: https://github.com/gentakojima/emojione-picker-ubuntu

As per their instructions, they currently use a `.deb` for installation.

**Comment by worldofpeace** [comment]
interesting, I think this application predates gtk having an emoji picker.

Not sure if it's right for inclusion in nixpkgs, last commit was a while ago.

**Comment by tycho01** [comment]
Fair enough. For context, I'm currently on KDE, which appears not to have one built-in.

**Comment by elmarsto** [comment]
still important to me

**Comment by elmarsto** [comment]
Currently cannot use KDE emoji picker because this ticket is open: https://github.com/NixOS/nixpkgs/issues/99678

Was surprised not to have emojione picker as failover

**Comment by jtojnar** [comment]
Ibus itself has an emoji picker so you can enable ibus and then use <kbd>ctrl</kbd>-<kbd>shift</kbd>-<kbd>e</kbd>.

**Comment by elmarsto** [comment]
This is great, thanks! I had no idea.

**Comment by SigmaSquadron** [comment]
Thanks for opening this package request. Unfortunately, in order to mitigate the unsustainable growth of unmaintained packages, Nixpkgs is no longer using package requests to coordinate new package additions.

That said, **we encourage you to [contribute](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#quick-start-to-adding-a-package) this package as a Pull Request yourself** - should you need any help during packaging, feel free to reach out on [Matrix](https://matrix.to/#/#dev:nixos.org) or [Discourse](https://discourse.nixos.org/c/dev/14). You are also welcome to open custom _Tracking_ issues if you would like the assistance of other volunteers in packaging a software ecosystem.

This change has the benefit of decreasing future maintainer burden on the long term, and was implemented after several contributors reached a consensus. You can read more about this on Pull Request [#391926](https://www.github.com/NixOS/nixpkgs/pull/391926) and Issue [#425040](https://www.github.com/NixOS/nixpkgs/issues/425040).
